### PR TITLE
Add `stripe` to payment methods

### PR DESCRIPTION
**Pull request by elia**
## Summary

- Fixes https://github.com/solidusio/solidus_stripe/issues/276
- ~~Blocked by https://github.com/solidusio/solidus/pull/5012~~
- Depends on https://github.com/solidusio/solidus/pull/5026 (first two commits)
- Depends on https://github.com/solidusio/solidus/pull/4989 (third commit)

## Checklist

Check out our [PR guidelines](https://github.com/solidusio/.github/blob/master/CONTRIBUTING.md#pull-request-guidelines) for more details.

The following are mandatory for all PRs:

- [ ] I have written a thorough PR description.
- [ ] I have kept my commits small and atomic.
- [ ] [I have used clear, explanatory commit messages](https://github.com/solidusio/.github/blob/main/CONTRIBUTING.md#writing-good-commit-messages).

The following are not always needed:

- 📖 I have updated the README to account for my changes.
- 📑 I have documented new code [with YARD](https://www.rubydoc.info/gems/yard/file/docs/Tags.md).
- 🛣️ I have opened a PR to update the [guides](https://github.com/solidusio/edgeguides).
- ✅ I have added automated tests to cover my changes.
- 📸 I have attached screenshots to demo visual changes.

### CODE REVIEW

File: core/lib/generators/solidus/install/app_templates/payment_method/stripe.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+if Bundler.locked_gems.dependencies['solidus_stripe']
+
+  generate 'solidus_paypal_commerce_platform:install'
```
Comment (kennyadsl) [inline core/lib/generators/solidus/install/app_templates/payment_method/stripe.rb:3]:
```suggestion
  generate 'solidus_stripe:install'
```

File: core/lib/generators/solidus/install/app_templates/payment_method/stripe.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+else
+
+  if @selected_frontend == 'starter'
+    bundle_command "add solidus_stripe --github='solidusio/solidus_stripe'"
+    generate "solidus_paypal_commerce_platform:install --migrate=#{!!options[:migrate]}"
```
Comment (kennyadsl) [inline core/lib/generators/solidus/install/app_templates/payment_method/stripe.rb:8]:
```suggestion
    generate "solidus_stripe:install --migrate=#{!!options[:migrate]}"
```

Comment (kennyadsl) [reply core/lib/generators/solidus/install/app_templates/payment_method/stripe.rb:8]: Why the `migrate` option?

**Review by kennyadsl — COMMENTED** [review]
I know it's draft, but I was taking a look and I left some preliminary comments.

File: core/lib/generators/solidus/install/app_templates/payment_method/stripe.rb (hunk 2 above)
Comment (elia) [reply core/lib/generators/solidus/install/app_templates/payment_method/stripe.rb]: Nevermind it was related to the wrong installer being triggered

**Review by kennyadsl — APPROVED** [review]

**Review by rainerdema — APPROVED** [review]
🚢

File: bin/sandbox [hunk 3]
```diff
@@ -93,12 +93,12 @@ fi
...
+#   $@
 
-echo "
-🚀 This app is intended for test purposes. If you're interested in running
-🚀 Solidus in production, visit:
-🚀 https://guides.solidus.io/developers/getting-started/first-time-installation"
+# echo "
+# 🚀 This app is intended for test purposes. If you're interested in running
+# 🚀 Solidus in production, visit:
+# 🚀 https://guides.solidus.io/developers/getting-started/first-time-installation"
```
Comment (waiting-for-dev) [inline bin/sandbox:104]: I think that changes are not intentional, right? :thinking:

**Review by waiting-for-dev — CHANGES_REQUESTED** [review]

File: bin/sandbox (hunk 3 above)
Comment (elia) [reply bin/sandbox:104]: Definitely not intentional, thanks for catching it!

It was needed in order to test the generator, so some food for thought, might make sense to back port the application generation cache we set up in solidus_stripe.

**Review by waiting-for-dev — APPROVED** [review]
:rocket:

**Pull request merged**